src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -158,7 +159,15 @@ private static void writeTablePropertiesRows(Sink sink)
         final Map<String, DetailNode> propertiesJavadocs = SiteUtil
                 .getPropertiesJavadocs(properties, currentModuleName, currentModuleFile);
 
-        for (String property : properties) {
+        final Set<String> validProperties = new LinkedHashSet<>(properties);
+
+        final String value = "tokens";
+
+        if (validProperties.remove(value)) {
+            validProperties.add(value);
+        }
```

Comment:
Make a method for this code with name `orderProperties`

Reply:
@romani Done